main.js

```diff
@@ -222,8 +213,8 @@ async function _checkStatus({
   // get JSON RevocationList
   const {credentialSubject: rl} = slCredential;
 
-  if(rl.type !== 'RevocationList2021') {
-    throw new Error('Revocation list type must be "RevocationList2021".');
+  if(rl.type !== 'StatusList2021') {
+    throw new Error('Revocation list type must be "StatusList2021".');
```

Comment:
Is using revocation list language here still proper?

Reply:
@dlongley just so you know some of the issues you have in this PR are from the multiple statuses PR I made so my apologies to @JSAssassin for mistakes I made there.

Reply:
No spanish inquisition here :P ... just noticing things we need fixed.

Reply:
Updated [b9a53d6](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/b9a53d64e2c4ae814291ee6aae9a9cd26d3758c6)